config-files/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### PR Checklist
+
+- [ ] JIRA Ticket(s) included in PR Title and are formatted correctly if applicable (JIRA-123)
+- [ ] PR Title has a short summary of what changes are included
+- [ ] PR Description below includes additional detail that isn't included in the title summary
+- [ ] Assign applicable leads to your PR for review
+
+### Additional PR Detail
+_Please provide additional detail of changes here._
```

Comment:
I like it, but thinking about it do we really need it at all? How about just leaving it out and saying `### Additional PR Details Below:` or something similar.